src/Npgsql/NpgsqlLargeObjectManager.cs

```diff
@@ -44,7 +44,7 @@ internal async Task<T> ExecuteFunction<T>(string function, bool async, params ob
             foreach (var argument in arguments)
                 command.Parameters.Add(new NpgsqlParameter { Value = argument });
 
-            return (T)(async ? await command.ExecuteScalarAsync() : command.ExecuteScalar());
+            return (T)(async ? await command.ExecuteScalarAsync() : command.ExecuteScalar())!;
```

Comment:
It is (since recently) possible to annotate generic type parameters, i.e. `Task<T?>` - this has the same effect as putting `[MaybeNull]` on a T directly. This is one of the important missing nullability things coming in .NET 5.0.

But in the specific case of this internal function, yeah, the expectation is for it to never return null so I think it's fine to leave it like this...